Source/CesiumRuntime/Private/CesiumGeoreference.cpp

```diff
@@ -586,6 +574,23 @@ void ACesiumGeoreference::_performOriginRebasing() {
   }
 }
 
+void ACesiumGeoreference::_updateGeoTransforms() {
+  glm::dvec3 center(0.0, 0.0, 0.0);
+  if (this->OriginPlacement == EOriginPlacement::CartographicOrigin) {
+    center = _geoTransforms.TransformLongitudeLatitudeHeightToEcef(glm::dvec3(
+        this->OriginLongitude,
+        this->OriginLatitude,
+        this->OriginHeight));
+  }
+
+  this->_geoTransforms = GeoTransforms(
```

Comment:
Is there a reason to create a new instance here? Originally, the `_geoTransforms` was supposed to be created only once, and only modified via the setters.

Reply:
Originally I was calling `setEllipsoid` followed by `setCenter`. But that recomputes the transforms twice. This way only does it once. We can probably just rid of the setters.

Reply:
The `setEllipsoid` and `setCenter` only update the transforms when the respective value actually *changed*. But even then, the call is not performance critical: It happens in `UpdateGeoreference`, and **if** this was called "in each tick", then 1. there would be something wrong, and 2. we'd have to look at all other implications that this may have: We don't know for sure *who* will listen to `OnGeoreferenceUpdated`, and *what* the listener will do, but I assume that whatever is done *there* will be more time critical than what is done in the `GeoTransforms`.

(And if we ran a benchmark in a profiler and noticed that more than 0.1% of the frame time is spent with computing that single matrix, we could still add a `setCenterAndEllipsoid` method and see whether this increased the overall performance, which I doubt).

One thought related to the `_geoTransforms` was **roughly**: There could/should be the "default `GeoTransforms` instance" that serves as the basis for the "default `Georeference`" (that is obtained with the `getDefault...` method). The "workflow" that was considered there was:

- Somebody obtains the default `Georeference`
- Somebory modifies it
- Somebody deletes it
- Some new default `Georeference` is created. It should have the state of the default one that was previously deleted (which was stored in the global, undeletable "default `GeoTransforms`").

(The `_geoTransforms` would have to be some `shared_ptr` for that to work, but that's an implementation detail). 

However, I'm fine with either solution, because the behavior of the  `getDefault...` method in view of deletions, undos, etc. has not been thought through in all depth.